wasm/binary/import.go

```diff
@@ -47,3 +47,25 @@ func decodeImport(r *bytes.Reader) (i *wasm.Import, err error) {
 	}
 	return
 }
+
+// encodeImport returns the wasm.Import encoded in WebAssembly 1.0 (MVP) Binary Format.
+//
+// See https://www.w3.org/TR/wasm-core-1/#function-types%E2%91%A4
+func encodeImport(i *wasm.Import) []byte {
+	data := encodeSizePrefixed([]byte(i.Module))
+	data = append(data, encodeSizePrefixed([]byte(i.Name))...)
+	data = append(data, i.Kind)
+	switch i.Kind {
+	case wasm.ImportKindFunc:
+		data = append(data, leb128.EncodeUint32(i.DescFunc)...)
+	case wasm.ImportKindTable:
+		panic("TODO: encodeImportKindTable")
```

Comment:
reminder: I have these TODO as they aren't implemented in the text format, yet. all progress in the text format should happen with a removed TODO and another line in example.wat